qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -37,26 +37,7 @@ You can download an example sample information file and prep information file fr
-| Value                                | Description                                                                                                                                      |
-+======================================+==================================================================================================================================================+
-| ``not applicable``                   | metadata information is inappropriate to report, can indicate that the standard itself fails to model or represent the information appropriately |
-+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``not available: not collected``     | metadata information of an expected format was not given because it has not been collected                                                       |
-+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``not available: to be reported``    | metadata information of an expected format was not given, a value will be given at the later stage                                               |
-+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``not available: restricted access`` | metadata information exists but can not be released openly                                                                                       |
-+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``obscured``                         | metadata information has been provided deliberately with low precision                                                                           |
-+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
-| ``temporarily obscured``             | metadata information has been provided deliberately with low precision, a higher precision value will be provided at a later stage               |
-+--------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------+
-
-The last two, which consider level of precision, will be used, for example, for geo-references (latitude, longitude and country/sea).
+For the latest definitions visit the `EBI/ENA Missing value reporting <http://www.ebi.ac.uk/ena/about/missing-values-reporting>`__.
```

Comment:
Do you think it is worth listing here how we actually write them to detect them? I.e. 'Not applicable', 'Missing: Not collected', 'Missing: Not provided', 'Missing: Restricted access'.